src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -260,6 +258,7 @@ Name|Type|Format|Description
 `expenseType`|`object`|[`Expense Type`](#schema-expensetype)|The expense type of the entry. **Required** for expected expenses, automatically set for segments depending on the `SegmentType` code.
 `href`|`string`|[RFC 3986]|Hyperlink to the resource for this Request entry.
 `id`|`string`|-|The unique identifier of the expected expense entry.
+`isPaymentCardRelated`|`boolean`|-|Indicates if the expected expense entry has the `Is Payment Card Related` field enabled. Value is `true` when the end user has selected this field for this expected expense entry.
```

Comment:
seems to be similar everywhere except for [ConcurRequest.swagger2.json](https://github.com/SAP-docs/preview.developer.concur.com/pull/1100/files#diff-6717e90089037c3a139c2e3e0fafed032a3cc61d11d14105f75d9929122fec7d) line 4796